apps/src/lib/kits/maker/CircuitPlaygroundBoard.js

```diff
@@ -191,22 +185,15 @@ export default class CircuitPlaygroundBoard extends EventEmitter {
   }
 
   digitalRead(pin, callback) {
-    return this.fiveBoard_.digitalRead(pin, callback);
+    this.fiveBoard_.digitalRead(pin, callback);
```

Comment:
One interface change: `digitalRead` and `analogRead` no longer return anything (the callback should be used to retrieve the value).  These were exposing the component for chaining, which was not meant to be part of our interface.